### PR TITLE
UIFC-24: switch to find-organization

### PR DESCRIPTION
**Pull request by elsenhans**

### CODE REVIEW

File: src/components/FindOrganization/FindOrganization.js [hunk 1]
```diff
@@ -14,97 +14,97 @@ import {
...
+        id="clickable-find-organization"
         {...this.props}
-        searchLabel="Vendor look-up"
+        searchLabel="Organization look-up"
         marginTop0
         searchButtonStyle="default"
-        dataKey="vendor"
-        selectVendor={this.selectVendor}
+        dataKey="organization"
+        selectOrganization={this.selectOrganization}
```
Comment (rchr) [inline src/components/FindOrganization/FindOrganization.js]: I get an error when selecting an organization:
```
Uncaught TypeError: this.props.selectVendor is not a function
    at OrganizationSearchModal.passVendorOut (OrganizationSearchModal.js:43)
    at OrganizationSearchModal.passVendorOut (react-hot-loader.development.js:661)
    at Object.onSelectRow [as onRowClick] (SearchAndSort.js:439)
    at MCLRenderer.handleRowClick (MCLRenderer.js:443)
    at onClick (MCLRenderer.js:901)
    at HTMLUnknownElement.callCallback (react-dom.development.js:149)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:199)
    at invokeGuardedCallback (react-dom.development.js:256)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:270)
    at executeDispatch (react-dom.development.js:561)

```
ui-plugin-find-organization still uses the property 'selectVendor'. It was not renamed to 'selectOrganization' (https://github.com/folio-org/ui-plugin-find-organization/blob/master/OrganizationSearch/OrganizationSearchModal.js#L14).

```suggestion
        selectVendor={this.selectOrganization}
```

File: package.json [hunk 2]
```diff
@@ -40,7 +42,7 @@
     "redux-form": "^7.0.3"
   },
   "peerDependencies": {
-    "@folio/stripes": "^2.0.0",
+    "@folio/stripes": "^2.3.0",
```
Comment (aliaksei-chumakou) [inline package.json]: that looks good, but maybe there is a possibility to use a newer version, https://github.com/folio-org/stripes/blob/master/doc/stripes-framework.md#upgrading-to-v26

File: translations/ui-finc-config/en_US.json [hunk 3]
```diff
@@ -1,63 +1,77 @@
 {
-    "meta.title": "Finc Config",
```
Comment (aliaksei-chumakou) [inline translations/ui-finc-config/en_US.json]: I'm not sure if devs are required to touch en_US.json, @dltj might provide a valuable input on this

**Comment by elsenhans** [comment]
Resolved the Review comments and checking out a vendor/organization is working

**Pull request merged**